src/imap.cpp

```diff
@@ -27,16 +31,16 @@ void IMapHandler::init(){
 /*
 Writes the IMap to the block device.
 */
-void IMapHandler::write(BlockDevice bd){
-    bd.write(I_MAP_FIRST_BLOCK, (char *) iMapStruct);
+void IMapHandler::write(BlockDevice* bd){
```

Comment:
BlockDevice mit oder ohne Pointer sollte überall vereinheitlicht werden

Reply:
Ja aber in net anderen branch dann